### PR TITLE
Get Hand Location Proc

### PR DESCRIPTION
**Pull request by Daxz0**
Gets the hand location of the target.

### CODE REVIEW

**Comment by acikek** [comment]
You could probably do without the header comment here

File: plugins/Denizen/scripts/Procs/get_hand_loc.dsc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+####################################
+
+
+
+get_hand_loc:
+    type: procedure
+    definitions: player
+    debug: false
+    script:
+        - define hand <[player].main_hand>
```
Comment (creaturesofhabit) [inline plugins/Denizen/scripts/Procs/get_hand_loc.dsc]: unused definition, can be removed

**Comment by acikek** [comment]
Also I'd prefer if the directories were lowercase

**Comment by Daxz0** [comment]
fixed

**Pull request merged**